build/common.sh

```diff
@@ -472,11 +488,8 @@ function kube::build::build_image() {
   kube::version::save_version_vars "${build_context_dir}/kube-version-defs"
 
   cp build/build-image/Dockerfile ${build_context_dir}/Dockerfile
-  if kube::build::is_osx; then
-    sed -i "" "s/KUBE_BUILD_IMAGE_CROSS/${KUBE_BUILD_IMAGE_CROSS}/" ${build_context_dir}/Dockerfile
```

Comment:
Oh I see.  That's cute, OSX.   Better to not dup the regex.  Do something like ixdy mentions above.